tests/test_repository.py

```diff
@@ -2527,6 +2535,7 @@ def test_set_commit_metrics():
         "blank_min": 0.0,
         "blank_total": 1.0,
     }
+
```

Comment:
Unrelated change. Could you revert this to keep it consistent with the rest of the file?